src/clj/game/cards/operations.clj

```diff
@@ -2994,20 +2994,18 @@
 
 (defcard "Under the Bus"
   {:on-play
-   {:req (req (and (last-turn? state :runner :accessed-cards)
-                   (not-empty (filter
-                                #(and (resource? %)
-                                      (has-subtype? % "Connection"))
-                                (all-active-installed state :runner)))))
+   {:req (req (last-turn? state :runner :accessed-cards))
     :prompt "Choose a connection to trash"
     :choices {:card #(and (runner? %)
                           (resource? %)
                           (has-subtype? % "Connection")
                           (installed? %))}
     :msg (msg "trash " (:title target) " and take 1 bad publicity")
     :async true
-    :effect (req (wait-for (trash state side target {:cause-card card})
-                           (gain-bad-publicity state :corp eid 1)))}})
+    :effect (req (if target (wait-for (trash state side target {:cause-card card})
```

Comment:
When would target not exist?

Reply:
Woops, I left this in from a different implementation - should be unchanged from original effect now.